tests/constants/datumaro_components.py

```diff
@@ -0,0 +1,6 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
Let's rename `constants` to something more descriptive.

Reply:
renamed